src/dxh.py

```diff
@@ -109,8 +133,11 @@ def evaluate_function_at_points(
         padded_points = np.zeros(points.shape[:-1] + (3,))
         padded_points[..., : points.shape[-1]] = points
         points = padded_points
-    tree = dolfinx.geometry.BoundingBoxTree(mesh, mesh.geometry.dim)
-    cell_candidates = dolfinx.geometry.compute_collisions(tree, points)
+    tree = bb_tree(mesh, mesh.geometry.dim)
+    cell_candidates = compute_collisions_points(tree, points)
+    # TODO: when dropping support for DOLFINx v0.6, replace the above two lines
+    # with the following for tidier namespace use.
```

Comment:
```suggestion
    # with the full namespace `dolfinx.geometry` for tidier namespace use.
```
(`black` ate my commented out lines 😭 )